pkg/brokerapi/service_instance.go

```diff
@@ -31,16 +31,9 @@ type ServiceInstance struct {
 	Parameters map[string]interface{} `json:"parameters, omitempty"`
 }
 
-// LastOperation represents the state of a discrete action that the broker is
-// completing asynchronously
```

Comment:
There wasn't a fantastic reason, other than it made sense to me to put the Create/Update/Delete Request/Response types up first because they're the main bread-&-butter, and then have the polling stuff be at the bottom. And I was thinking the same with the client itself, which is why I put `pollBroker` at the bottom of `open_service_broker_client.go`